jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/ImageMetadataOutput.java

```diff
@@ -62,7 +68,11 @@ public static ImageMetadataOutput fromJibContainer(JibContainer jibContainer) {
     String image = jibContainer.getTargetImage().toString();
     String imageId = jibContainer.getImageId().toString();
     String imageDigest = jibContainer.getDigest().toString();
-    return new ImageMetadataOutput(image, imageId, imageDigest);
+
+    // Make sure tags always appear in a predictable way, by sorting them into a list
+    List<String> tags = jibContainer.getTags().stream().sorted().collect(Collectors.toList());
```

Comment:
I think Guava's [`ImmutableList.sortedCopyOf()`](https://guava.dev/releases/28.2-jre/api/docs/com/google/common/collect/ImmutableList.html#sortedCopyOf-java.lang.Iterable-) is a little more compact, and immutable?

Reply:
Good idea, updated!